dev/bots/prepare_package.dart

```diff
@@ -200,15 +204,32 @@ class ArchiveCreator {
     _processRunner.environment['PUB_CACHE'] = path.join(flutterRoot.absolute.path, '.pub-cache');
   }
 
+  /// The platform to use for the environment and determining which platform we're running on.
   final Platform platform;
+
+  /// The branch to build the archive for.
   final Branch branch;
+
+  /// The git revision hash to build the archive for.
```

Comment:
does this one have to be part of [branch]?

Reply:
Yes, the git revision has to be available in the branch.  I clone the repo with the name "branch", and then reset to this revision.  I could just clone the branch, but there's no guarantee that the HEAD will be the right revision, so I need both.  I'll add that info to the docs.